src/follower.py

```diff
@@ -98,17 +100,26 @@ def convert_camera_angle(self):
         else:
             self._turn_angle = 90
 
-
     """
     Resets the camera to the default position.
 
     The default position is tilted to 120 degrees and panned to 90 degrees.
     """
+    
     def reset_camera(self):
         self._camera.turn_down(120)
         self._camera.pan_servo.write(90)
 
+    """
+    Definitions for turning the camera left and right
+    """
 
+    def _turn_camera_left(self, angle):
+        self._camera.turn_left(angle)
```

Comment:
camera.turn_left and camera.turn_right do not take angles. They take steps. Please check the camera script for more details on this.

Reply:
corrected parameter names